### ISSUE TITLE
Importing ActiveStorage Images

### ISSUE THREAD

**Issue by kadirpace** [zdennis/activerecord-import#661]
I am trying to import products and save images through remote urls. Import works well, but I can't manage it to make it work work ActiveStorage images.

Is this supported in anyway? This is what I have

```
    product = Product.new(
        name: row['name'],
        sku: row['sku'],
        brand_id: row['brand']
    )
    product.image.attach(io: parsed_image_url, filename: row['sku']+ '.jpg', content_type: 'image/jpg')
    products << product
...
    importing = Product.import products, recursive: true
```
when I check the development logs, I don't see any actions for images.

**Comment by jkowens** [comment]
I've never tested with ActiveStorage specifically. I'd say it's very likely that support for ActiveStorage attachments would need to be added. Just to verify, you are using Postgres correct? The recursive option is only supported for Postgres databases.

**Comment by kadirpace** [comment]
hi @jkowens, support would be nice. I believe many people are using Activestorage option these days.
yes, I am using Postgres database.

**Comment by jkowens** [comment]
I agree, if anyone can get a PR together for this in the near term that would be appreciated 👍
